app/code/Magento/Store/Test/Unit/Model/Address/RendererTest.php

```diff
@@ -41,7 +41,11 @@ protected function setUp()
                 return implode("\n", $data['variables']);
             });
 
-        $this->model = new Renderer($eventManager, $filterManager);
+        $template = "{{var name}}\n{{var street_line1}}\n{{depend street_line2}}{{var street_line2}}\n{{/depend}}" .
```

Comment:
Can we have it as a `DEFAULT_TEMPLATE` constant in original class maybe? And then just specify it as a default value of `$template` argument.

Reply:
I've updated the pull request to include the constant.